RUFAS/biophysical/field/crop/crop.py

```
@@ -71,7 +70,7 @@ class Crop:
 
     """
 
-    def __init__(self, crop_data: Optional[CropData] = None):
+    def __init__(self, crop_data: CropData | None = None):
```

Ruff (0.15.15)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)